infinispan-embedded/src/test/java/org/hibernate/ogm/datastore/infinispan/utils/InfinispanTestHelper.java

```diff
@@ -48,6 +56,24 @@
 
 	private static final Log log = LoggerFactory.make( MethodHandles.lookup() );
 
+	@Override
+	public void prepareDatabase(SessionFactory sessionFactory) {
+		super.prepareDatabase( sessionFactory );
+		InfinispanEmbeddedDatastoreProvider provider = getProvider( sessionFactory );
+		LocalCacheManager<?, ?, ?> lcm = provider.getCacheManager();
+		EmbeddedCacheManager ecm = lcm.getCacheManager();
+		registerStoredProcedures( ecm );
+	}
+
+	private void registerStoredProcedures(EmbeddedCacheManager ecm) {
+		Cache<Object, Object> storedProcedures = ecm.getCache( "___stored_procedures" );
+		storedProcedures.put( Car.SIMPLE_VALUE_PROC, SimpleValueProcedure.class.getName() );
+		storedProcedures.put( Car.RESULT_SET_PROC, ResultSetProcedure.class.getName() );
+		storedProcedures.put( InvalidStoredProcedure.INVALID_STORED_PROCEDURE_NAME, InvalidStoredProcedure.class.getName() );
+		// See NamedParametersStoredProcedureCallTest#testExceptionWhenProcedureFails
+		storedProcedures.put( "exceptionalProcedure", ExceptionalProcedure.class.getName() );
```

Comment:
Why don't use MockStoredProcedureDialect#EXCEPTIONAL_PROCEDURE_NAME constants?

Reply:
Good question, I did it initially but then I head an exception when running the build.
I didn't want to spend too much time on it and in the end I decided to keep it like this.

I would prefer to merge this anyway and figure out later what happened.